diff -ru gmp-4.3.2/tests/mpz/t-scan.c gmp-4.3.2-tscan/tests/mpz/t-scan.c
--- gmp-4.3.2/tests/mpz/t-scan.c	2010-01-07 20:09:03.000000000 +0000
+++ gmp-4.3.2-tscan/tests/mpz/t-scan.c	2013-12-10 11:52:05.000000000 +0000
@@ -79,7 +79,7 @@
 
               for (isize = 0; isize <= size; isize++)
                 {
-                  for (oindex = 0; oindex <= numberof (offset); oindex++)
+                  for (oindex = 0; oindex < numberof (offset); oindex++)
                     {
                       o = offset[oindex];
                       if ((int) isize*GMP_NUMB_BITS < -o)
